FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Bryan Hoke on 8/11/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
`ManagedCache` is not supposed to be subclassed, so it should be marked as `final`:
```suggestion
final class ManagedCache: NSManagedObject {
```